src/helpers/helpers.ts

```diff
@@ -0,0 +1,21 @@
+import { Todo, TodoCompleteStatus } from '../types/Todo';
+
+export const todoFilteredCompleted = (
+  (todoCompleteStatus: TodoCompleteStatus, todos: Todo[]) => {
+    switch (todoCompleteStatus) {
+      case TodoCompleteStatus.Active:
+        return todos.filter(todo => !todo.completed);
+
+      case TodoCompleteStatus.Completed:
+        return todos.filter(todo => todo.completed);
+
+      default:
+        return todos;
+    }
+  });
+
+export const getTodoCompletedId = (todos: Todo[]) => {
+  const filteredTodos = todos.filter(todo => todo.completed);
```

Comment:
You could pass the value from `useMemo`

Reply:
Sorry, but can you discribbe how to do it, because it is not clear for me,thx